pkg/image/api/v1/types.go

```diff
@@ -57,41 +57,44 @@ type ImageLayer struct {
 // ImageSignature holds a signature of an image. It allows to verify image identity and possibly other claims
 // as long as the signature is trusted. Based on this information it is possible to restrict runnable images
 // to those matching cluster-wide policy.
-// There are two mandatory fields provided by client: Type and Content. They should be parsed by clients doing
-// image verification. The others are parsed from signature's content by the server. They serve just an
-// informative purpose.
+// Mandatory fields should be parsed by clients doing image verification. The others are parsed from
+// signature's content by the server. They serve just an informative purpose.
 type ImageSignature struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	kapi.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
 	// Required: Describes a type of stored blob.
-	Type string `json:"type" protobuf:"bytes,1,opt,name=type"`
+	Type string `json:"type" protobuf:"bytes,2,opt,name=type"`
```

Comment:
Can I safely assume that signatures haven't been really used anywhere until now to bump these numbers? Or do I have to increase the index of `ObjectMeta` instead?